automation/test.sh

```diff
@@ -436,11 +436,9 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
     label_filter='(sig-compute-migrations && !(GPU,VGPU))'
   elif [[ $TARGET =~ sig-compute-serial ]]; then
     export KUBEVIRT_E2E_PARALLEL=false
-    ginko_params="${ginko_params} --focus=\\[Serial\\]"
-    label_filter='((sig-compute) && !(GPU,VGPU,sig-compute-migrations))'
+    label_filter='((sig-compute,Serial) && !(GPU,VGPU,sig-compute-migrations))'
```

Comment:
Should this be?
```suggestion
    label_filter='((sig-compute && Serial) && !(GPU,VGPU,sig-compute-migrations))'
```

Reply:
Interesting, I'm honestly not sure why this works as is

Reply:
I think `,` is an OR operation so it will run all sig-compute and all serial ... it will take a while.

Reply:
You're right my bad